src/components/media/Media.scss

```diff
@@ -18,6 +18,7 @@
 // the main content, e.g. title and message
 .bdl-Media-body {
     flex: 1 1 100%;
+    overflow: hidden;
```

Comment:
Is this always true for every Media object?

Reply:
That's a good point, let me put this style under `.bcs-Task`